src/MergeContext.js

```diff
@@ -856,10 +513,302 @@ class MergeContext {
         return true;
     }
 
+    _toString() {
+        let str = this._debugString();
+        if (this._tagSha !== null)
+            str += ", tag: " + this._tagSha.substr(0, this._shaLimit);
+        return str + ")";
+    }
+} // MergeContext
+
+// Starts PR processing.
+// Performs different checks over the PR and creates staged commit
```

Comment:
I would generalize the second sentence into something like "Performs all PR merge steps up to and including the staged commit creation".

Reply:
Done.